nilearn/plotting/js_plotting_utils.py

```diff
@@ -22,39 +23,55 @@
 
 MAX_IMG_VIEWS_BEFORE_WARNING = 10
 
+NIIVUE_VERSION = "0.27.0"
 
-def add_js_lib(html, embed_js=True):
+LIBRARY_URL = {
+    "plotly": "https://cdn.plot.ly/plotly-gl3d-latest.min.js",
+    "jquery": (
+        "https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"
+    ),
+    "niivue": f"https://unpkg.com/browse/@niivue/niivue@{NIIVUE_VERSION}/dist/index.js",
```

Comment:
@hanayik 

Is this what you meant by using unpkg?

https://github.com/nilearn/nilearn/pull/3729#discussion_r1918426397

Reply:
Actually, `"https://unpkg.com/browse/@niivue/niivue@{NIIVUE_VERSION}/build/index.js"` would be better. I gave the wrong path earlier.